engine/src/main/java/org/terasology/rendering/cameras/OculusStereoCamera.java

```diff
@@ -172,6 +172,9 @@ public Matrix4f getInverseViewProjectionMatrix() {
         return null;
     }
 
+    /**
+     * @deprecated
```

Comment:
I agree with Martin, you might fix the checkstyle warning this way but you'd also hide the longer term issue of eliminating these methods altogether.